InOut/rtjack.c

```diff
@@ -1405,7 +1405,7 @@ static int32_t midi_out_open(CSOUND *csound, void **userData,
     pm =
       (RtJackMIDIGlobals*) csound->QueryGlobalVariableNoCheck(csound,
                                                               "_rtjackMIDIGlobals");
-    sprintf(clientName, "%s_out", pm->clientName);
+    snprintf(clientName, MAX_NAME_LEN+4, "%s_out", pm->clientName);
```

Comment:
Checking, should this be +5 to account for _out and NULL terminator?

Reply:
As before, there is only 4 added bytes in the array.